profiles/tests/test_gql_my_profile_query.py

```diff
@@ -634,25 +646,42 @@ def mock_return(*_, **__):
         {
             myProfile {
                 loginMethods
+                availableLoginMethods {
+                    method
+                    createdAt
+                    userLabel
+                }
             }
         }
     """
     executed = user_gql_client.execute(
         query, auth_token_payload={"amr": amr_claim_value}, service=service
     )
     assert "errors" not in executed
-    assert set(executed["data"]["myProfile"]["loginMethods"]) == {
+    assert executed["data"]["myProfile"]["loginMethods"] == [
```

Comment:
Can the order change here or is it always the same? And if it can change, does the order matter? The reason I originally used `set` here instead of a `list` was to ignore the order of the items and just compare the values (because `["foo", "bar"] != ["bar", "foo"]`)

Reply:
The order should be stable at least in the tests since database doesn't affect the order. Order doesn't matter, so in that sense it could be a `set`. Though a `set` could potentially hide a value existing multiple times in the data.

Reply:
Yeah, the potential duplicates were also a reason why I used sets quite a lot, as you probably noticed. My reasoning back then was that the duplicates wouldn't probably really matter, so I used sets for deduplication.